resources/tag_models/event.json

```diff
@@ -0,0 +1,29 @@
+{
+  "tag": {
+	"api_key": "", // String - API key - Make sure the request is from a valid client (idea nicked from Keen.io) useful if a page gets copied onto a Russian website and creates noise
+	"version": "", // Version of the tracking client e.g. "JS v1.2"
+	"type": "event", // Type of request (page or event)
+	"id": "", // ID of this request, must be unique
```

Comment:
But why not use an ID? It's more accurate and saves having to send through a lot of data with each request. WebApp, for example, won't be keen on unnecessarily large payloads.

In your suggestion, we'd have to send through the xpath of the element, plus repeat info about the page and other things? Why is that better than an id?

Also, you assume events will always be from a html page. Already we're talking about collecting email events. Opens and clicks are a good example of needing a hierarchical model too. A parent/child ID is the only consistent way.